extensions/wikia/UserProfilePageV3/Masthead.class.php

```diff
@@ -122,7 +122,7 @@ public function getDefaultAvatars( $thumb = "" ) {
 		wfProfileIn( __METHOD__ );
 
 		$this->mDefaultAvatars = array();
-		$images = getMessageForContentAsArray( 'blog-avatar-defaults' );
+		$images = getMessageAsArray( 'blog-avatar-defaults' );
```

Comment:
`getMessageAsArray` is not "ForContent" like the old method was. `getMessageForContentAsArray` should be updated to use the new methods like `getMessageAsArray` uses and make use of  `->inContentLanguage()` on the Message object so it has the same behaviour.